projects/portland-metro/.env

```diff
@@ -1,2 +1,3 @@
 COMPOSE_PROJECT_NAME=pelias
 DATA_DIR=/tmp/pelias/portland-metro
+DOCKER_USER=1000
```

Comment:
oops I missed this in review, fixed in https://github.com/pelias/docker/pull/224